premailer/premailer.py

```diff
@@ -395,8 +397,11 @@ def transform(self, pretty_print=True, **kwargs):
         # and a long list of elements
         for _, element in elements.items():
             final_style = merge_styles(element['item'].attrib.get('style', ''),
-                                       element['style'], element['classes'])
-            element['item'].attrib['style'] = final_style
+                                       element['style'], element['classes'],
+                                       unset_removes_attribute=self.unset_removes_attribute)
```

Comment:
Can you please rewrite these lines to be:
```python
final_style = merge_styles(
    element['item'].attrib.get('style', ''),
    element['style'], 
    element['classes'],
    unset_removes_attribute=self.unset_removes_attribute,
)
```
That kind of formatting has the advantage that it's indentation for line 2 and 3 isn't dependent of the length of the variable it assigns to (e.g. `final_style`).
Also, note the trailing comma on the last parameter.